quesma/ab_testing/controller/controller.go

```diff
@@ -0,0 +1,121 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package controller
```

Comment:
There is too much from `MVC`, but it has a different context.

Maybe name `coordinator`?